samples/exemplarchat/README.md

```diff
@@ -2,3 +2,21 @@
 
 This is a sample app that makes use of Misk. It demonstrates both web sockets and Misk's recommended
 patterns for embedded web UIs.
+
+## Usage
+
+1. Build the React app in the `web` directory.
+    ```
+    cd samples/exemplarchat/web
+    yarn
```

Comment:
Do you have a preference for `npm` over `yarn` here?

Reply:
@adrw do you have a preference?

Reply:
`npm`!